package.json

```diff
@@ -76,6 +76,7 @@
 		"@wordpress/npm-package-json-lint-config": "file:packages/npm-package-json-lint-config",
 		"@wordpress/postcss-themes": "file:packages/postcss-themes",
 		"@wordpress/scripts": "file:packages/scripts",
+		"accessible-autocomplete": "1.6.2",
```

Comment:
I think this should be defined as a `dependencies` of `packages/editor/package.json`, not here.

Reply:
moved in https://github.com/WordPress/gutenberg/pull/7385/commits/27cedb3f3919ecb40e9b1dde34d993b15fac4a9b